lib/screens/v3_api.ex

```diff
@@ -69,11 +69,18 @@ defmodule Screens.V3Api do
   end
 
   defp build_url(route, params) when map_size(params) == 0 do
-    base_url() <> route
+    base_url()
+    |> URI.parse()
+    |> URI.merge(route)
+    |> URI.to_string()
   end
 
   defp build_url(route, params) do
-    "#{base_url()}#{route}?#{URI.encode_query(params)}"
+    base_url()
+    |> URI.parse()
+    |> URI.merge(route)
+    |> URI.to_string()
+    |> then(&"#{&1}?#{URI.encode_query(params)}")
```

Comment:
why these updates?

Reply:
From the PR description:
> Due to recent changes that came from Screenplay, I needed to adjust the V3Api module to standardize how it builds the API's url. It now properly forms the url even when the base URL doesn't have a trailing `/`.

I started getting an `:nxdomain` error in all the V3 API requests from Screens when I started it up after setting new env vars for the [values that Screenplay now requires](https://github.com/mbta/screenplay/pull/206).

I guess both apps check the same env var for the V3 API base URL, and one (Screens) [expects the base URL to end with a `/`](https://github.com/mbta/screens/blob/master/lib/screens/v3_api.ex#L76) while the other (Screenplay) [does not](https://github.com/mbta/screenplay/blob/master/lib/screenplay/v3_api.ex#L69). By using functions from the `URI` module, we can handle both cases in an ISO-standard way instead of doing raw string concatenation and having the app break when a trailing slash is present/missing.

Reply:
Actually! I just noticed a bug in Screenplay code.
- If we make a V3 API request without any query params, it does not put a slash between the base URL and the resource. [link to line](https://github.com/mbta/screenplay/blob/master/lib/screenplay/v3_api.ex#L65)
- If we make a V3 API request with query params, it _does_ put a slash between the base URL and the resource. [link to line](https://github.com/mbta/screenplay/blob/master/lib/screenplay/v3_api.ex#L69)

We should probably use the `URI` functions over there as well to avoid these kinds of bugs with constructing URLs.